README.md

```diff
@@ -11,6 +11,66 @@ for more information about JW Platform API.
 
 Java 8 and later.
 
+
+# V2 Client ##
+
+## Install With Maven:
+
+Add this dependency to your project's POM:
+
+```xml
+<dependency>
+  <groupId>com.jwplayer</groupId>
+  <artifactId>jwplatform</artifactId>
+  <version>1.0.0</version>
```

Comment:
the v1 client is fully function on v1.0.0 so I'm fine just moving the Install to a section above this. We don't need to suggest v0.4.0 for v1 client since it works on v1.0.0.